test/unittest/dds/subscriber/DataReaderTests.cpp

```diff
@@ -64,9 +65,20 @@
 #include <fastdds/rtps/transport/test_UDPv4TransportDescriptor.h>
 #include <fastrtps/xmlparser/XMLProfileManager.h>
 
+#include <asio.hpp>
+
+#if defined(_WIN32)
```

Comment:
Same comment as in the other file. Maybe either add the winrt check to keep it in line with what SystemInfo uses or use that one instead.

Reply:
Done in d68b310